Formula/kibana.rb

```diff
@@ -19,12 +19,17 @@ class Kibana < Formula
   end
 
   resource "yarn" do
-    url "https://yarnpkg.com/downloads/1.16.0/yarn-v1.16.0.tar.gz"
-    sha256 "df202627d9a70cf09ef2fb11cb298cb619db1b958590959d6f6e571b50656029"
+    url "https://yarnpkg.com/downloads/1.21.1/yarn-v1.21.1.tar.gz"
+    sha256 "d1d9f4a0f16f5ed484e814afeb98f39b82d4728c6c8beaafb5abc99c02db6674"
   end
 
   def install
     resource("node").stage do
+      # Fixes detecting Apple clang 11. (this is the patch that's applied in the node@10 formula)
+      inreplace "configure.py" do |s|
+        s.gsub! 'cc, r"(^Apple LLVM version) ([0-9]+\.[0-9]+)")', 'cc, r"(^Apple (?:clang|LLVM) version) ([0-9]+\.[0-9]+)")'
+      end
```

Comment:
There's a [patch](https://github.com/Homebrew/homebrew-core/blob/master/Formula/node@10.rb#L20) for Node 10 that causes [this build error](https://github.com/Homebrew/homebrew-core/pull/46998#issuecomment-557167583). Since this only uses `node@10` as a resource and not as a dependency, I just pulled in the patch manually (unless there's a better way to do this?)

Reply:
Maybe we can see if it can depend on the node formula?

Reply:
@SMillerDev we'd have to use `node@10` -- do you know if that installs the dependent formula onto the system, rather than just using it for build?

Reply:
You can use `depends_on "node@10" => :build`